rpc/serde/jsonrpc.go

```diff
@@ -138,3 +138,44 @@ func NewJsonRpcResponse[T ResponsePayload](requestId uint64, objectiveResponse T
 		Result:  objectiveResponse,
 	}
 }
+
+type JsonRpcGeneralRequest struct {
+	Jsonrpc string      `json:"jsonrpc"`
+	Id      uint64      `json:"id"`
+	Method  string      `json:"code"`
```

Comment:
Is this struct tag right?

Reply:
That is not correct. Good catch. Changed to "method"